CMakeLists.txt

```diff
@@ -233,34 +233,42 @@ if (BUILD_TESTS AND NOT SKIP_SRC)
 
   find_package (cgreen REQUIRED)
 
-  add_custom_target (tests DEPENDS
-                     array-test
-                     boreas-alivedetection-test
-                     boreas-cli-test
-                     boreas-error-test
-                     boreas-io-test
-                     boreas-ping-test
-                     boreas-sniffer-test
-                     compressutils-test
-                     cpeutils-test
-                     cvss-test
-                     hosts-test
-                     httputils-test
-                     json-test
-                     jsonpull-test
-                     logging-test
-                     logging-domain-test
-                     networking-test
-                     nvti-test
-                     openvasd-test
-                     osp-test
-                     passwordbasedauthentication-test
-                     test-hosts
-                     util-test
-                     version-test
-                     versionutils-test
-                     vtparser-test
-                     xmlutils-test)
+  set (TESTS
+      array-test
```

Comment:
Looks like the indent is one char too little here.

Reply:
It seems the formatting is not consistent in all cases in this file. Nevertheless I adapted the formatting as you suggested because it was completely different. I am going to look into an autoformatter tool in near future for cmake files.